src/components/Login/LoginWithPwd.tsx

```diff
@@ -110,6 +148,9 @@ const LoginWithPwd: React.FC<Props> = (props: Props) => {
             <Button variant="shareBtn" width="300px" marginTop="20px" type="submit">
               Continue Login
             </Button>
+            <Button variant="link" width="300px" marginTop="15px" onClick={handleForgotPwd}>
+              <Text fontSize="15px">Forgot password?</Text>
+            </Button>
```

Comment:
it can be replaced with 
```
<Button
    fontSize="15px"
    variant="link"
    width="300px"
    marginTop="15px"
    onClick={handleForgotPwd}
  >
    Forgot password?
  </Button>
```

Reply:
Understood! Thanks Shawn!